### DISCUSSION TITLE
Pyinstaller not working with Vosk

### DISCUSSION THREAD

**Discussion by arvindmn01** [pyinstaller/pyinstaller#7794] [PyInstaller] [answered]
I have created exe file using Pyinstaller. i am using vosk in the backend but when i open the exe file it gives me this error. 

![Screenshot 2023-07-28 145322](https://github.com/pyinstaller/pyinstaller/assets/100796728/5ae03550-79aa-430f-b9cd-29cdcaa1b94c)

**Comment by rokm** [comment, marked as answer]
If `vosk` is trying to add the directory via `os.add_dll_directory`, it likely has some DLLs that need to be collected... try adding `--collect-binaries vosk` to your PyInstaller command.
